test/e2e/egress_firewall.go

```diff
@@ -491,26 +490,8 @@ spec:
 		}
 
 		ginkgo.By("Creating the nodePort service")
-		npSpec := &v1.Service{
-			ObjectMeta: metav1.ObjectMeta{
-				Name: serviceName,
-			},
-			Spec: v1.ServiceSpec{
-				Type: v1.ServiceTypeNodePort,
-				Ports: []v1.ServicePort{
-					{
-						Port:       int32(servicePort),
-						NodePort:   int32(servicePort),
-						Name:       "http",
-						Protocol:   v1.ProtocolTCP,
-						TargetPort: intstr.FromInt(efPodPort),
-					},
-				},
-				Selector: endpointsSelector,
-			},
-		}
-		_, err = f.ClientSet.CoreV1().Services(f.Namespace.Name).Create(context.Background(), npSpec, metav1.CreateOptions{})
-		framework.ExpectNoError(err)
+		_, err = createServiceForPodsWithLabel(f, f.Namespace.Name, int32(servicePort), strconv.Itoa(efPodPort), "NodePort", endpointsSelector)
```

Comment:
I am confused because I see that this function receives a nodePort that is not being passed: https://github.com/tssurya/ovn-kubernetes/blob/af58536d749c2ad377e3e1a032375ffcee4dcb4c/test/e2e/e2e.go#L269-L270
```
func createServiceForPodsWithLabel(f *framework.Framework, namespace string, servicePort int32, targetPort string, nodePort int32, serviceType string, labels map[string]string) (string, error) {
```

Reply:
hmm where did you get this link from? I am also a bit confused.. perhaps the hash is wrong? can you double check. This PR https://github.com/ovn-org/ovn-kubernetes/pull/3481 is where I removed it.

Reply:
not sure anymore :P